tests/terminate/pos/Div.hs

```diff
@@ -0,0 +1,19 @@
+module Div where
```

Comment:
Could be helpful to start the test with a comment that makes explicit what is being tested.

Reply:
Isn't it implied by the location of the test that we are testing termination? Are we testing something else? As far as I can tell, we are not testing the `GHC.Internal.Real.div` assumptions.

Reply:
It is true that the reader might assemble the explanation. But something more explicit is cheap and can make it harder to get it wrong. Here's a try.

```suggestion
-- Tests that termination is detected when using div in the argument
-- of the recursive call.
module Div where
```